packages/api/src/functions/graphql.ts

```diff
@@ -10,7 +10,8 @@ import { setContext } from 'src/globalContext'
 
 export type GetCurrentUser = (
   decoded: AuthContextPayload[0],
-  raw: AuthContextPayload[1]
+  raw: AuthContextPayload[1],
+  event: AuthContextPayload[2]
```

Comment:
@peterp and @brentjanderson Naming the third param just event doesn't seem right since if in fact contains:

```
{ event, context }
```

@peterp  commented

>That seems 100% reasonable to me. It'll allow us to add more "options" without having to add more params.

Perhaps it is called `options` rather than `event`?

Reply:
`options` sounds fine to me, `input` or `signal` or `trigger` or `context` make sense too since we're basically describing what triggered getCurrentUser, and in what way.